app/models/lockstep/user.rb

```diff
@@ -3,6 +3,14 @@ class Lockstep::User < Lockstep::ApiRecord
   self.id_ref = "user_id"
   load_schema(Schema::UserAccount)
 
+  def self.invite(email_addresses)
+    emails = email_addresses.map { |email| { email: email } }
+    resp = resource.post('/invite', body: emails, params: {})
+    raise Lockstep::Exceptions::BadRequestError, 'Endpoint not found' if resp.code == '404'
```

Comment:
Bro we need to raise error for anything other than a 200. we shouldn't just scope it for 404.

Reply:
changes added [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/af1d66ab38cff7c2ac3e97c1f507db5d50971fa6)